agent/balrogagent/cmd.py

```diff
@@ -43,7 +44,14 @@
                     else:
                         logging.debug("Unknown change type!")
 
-                    # If it *is* ready, enact it!
+                    # If it *is* ready, check if all the required signoffs have
+                    # been obtained
+                    required_signoffs = change.get('required_signoffs') or {}
+                    signoffs = change.get('signoffs') or {}
+                    if not verify_signoffs(required_signoffs, signoffs):
```

Comment:
Can you add a log message here? Right now we have output like:
balrogagent_1  | 2017-02-23 14:37:19,906 - DEBUG - root.time_is_ready#22: Comparing time for change 1 (now: 1487860639.9067075, scheduled time: 1487860530.912)
balrogagent_1  | 2017-02-23 14:37:19,907 - DEBUG - root.run_agent#61: Change 1 is not ready

It logs the details of the time (which make it look it the change is ready), but then it says "not ready".